app/src/main/res/layout/component_collection_creation.xml

```diff
@@ -27,7 +27,7 @@
         android:textAppearance="@style/HeaderTextStyle"
         android:textColor="@color/neutral_text"
         android:textSize="20sp"
-        app:layout_constraintEnd_toStartOf="@+id/guideline"
+        app:layout_constraintEnd_toStartOf="@id/guideline"
```

Comment:
Maybe I'm looking it from the wrong angle, but since this is the first appearance of `guideline` in this xml, shouldn't this one be with the '+' and the others below without? It works though, so maybe `guideline` was already declared before in a different xml